src/components/Hero/hero.style.js

```diff
@@ -1,14 +1,13 @@
-import { Link } from 'react-router-dom'
-import styled from 'styled-components'
-import { Swiper, SwiperSlide } from 'swiper/react'
-
-export const H = {}
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import { Swiper, SwiperSlide } from "swiper/react";
+export const H = {};
 
 H.Swiper = styled(Swiper)`
   position: relative;
   display: flex;
   width: 100%;
-  height: 80vh;
+  height: 60vh;
```

Comment:
not sure in what resolution did you tested but in aniwatch.to the height looks nearly the same as in the old repo, 60 it's just to small

Reply:
![image](https://github.com/fiskryeziu/zoroClone/assets/108014780/464fce07-8f0d-417b-a186-17dd7ebca96e)
In aniwatch the trending section appears as whole and also the title and description cover almost all the height of  hero component.

Reply:
which resolution you have?  for e.g in my laptop I have 1367px and the height looks like this in aniwatch
![Screenshot (5)](https://github.com/fiskryeziu/zoroClone/assets/64495368/8f2cf8d1-9a79-4b60-9407-c5e6cd710e9a)

Reply:
mine is 1980*1080p. I have tried both on my laptop and external monitor. I still see the same. Nevertheless, I will definitely try to improve it as much as I can. You have done a really good job with this project. I'll try to come up with a solution to avoid compromising the gradient and still maintaining a good resolution on the picture

Reply:
I played around a bit with the resolution in aniwatch  for e.g I toggled the device toolbar and changed the window height, while changing I saw no changes in other  elements height or width. But when I change the height in old repo seems like the elements are reacting to that.

Reply:
it might be because of the % units